report-generator/src/test/java/com/teamscale/report/util/BashFileSkippingInputStreamTest.java

```diff
@@ -0,0 +1,27 @@
+package com.teamscale.report.util;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.jar.JarEntry;
+import java.util.jar.JarInputStream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class BashFileSkippingInputStreamTest {
+
+	@Test
+	void testBashFileJar() throws IOException {
```

Comment:
please also add a test for a normal jar to ensure this still works as before

Reply:
Sure